Tests/SwiftStdlibTests/ArrayExtensionsTests.swift

```diff
@@ -242,9 +242,11 @@ final class ArrayExtensionsTests: XCTestCase {
 
     func testGroupBy() {
         let array: [String] = ["james", "irving", "jordan", "jonshon", "iverson"]
-        let grouped = array.groupByKey { element -> String in
+
```

Comment:
Good catch? What happened here 🤔

Reply:
Not entirely sure, also check out the edit in the CHANGELOG, seemed messed up as well